templates/master.yaml.tmpl

```diff
@@ -1739,9 +1739,145 @@ storage:
           rules:
           - apiGroups:
             - ""
+            - apiregistration.k8s.io
+            - batch
+            - certmanager.k8s.io
+            - cluster.k8s.io
+            - clusterissuers
+            - coordination.k8s.io
+            - extensions
+            - issuers
+            - networking.k8s.io
+            - rbac.authorization
+            - scheduling.k8s.io
+            - storage.k8s.io
             resources:
+            - apiservices
+            - certificate
+            - componentstatus
+            - clusterroles
+            - clusterrolebindings
+            - clusters
+            - cronjobs
+            - daemonsets
+            - deployments
+            - endpoints
+            - ingresses
+            - leases
+            - machinedeployments
+            - namespaces
+            - networkpolicies
+            - nodeconfigs
             - nodes
+            - pods
+            - podsecuritypolicies
+            - priorityclasses
+            - replicasets
+            - roles
+            - rolebindings
+            - serviceaccounts
+            - services
+            - storageclass
+            verbs:
+            - get
+            - list
+          - apiGroups:
+            - ""
+            resources:
+            - configmaps
+            verbs:
+            - list
+          - apiGroups:
+            - ""
+            resources:
+            - configmaps
+            resourceNames:
+            - api-configmap
+            - app-operator
+            - aws-operator-configmap
+            - azure-operator-configmap
+            - bridge-operator-configmap
+            - cert-operator-configmap
+            - chart-operator
+            - cluster-operator-configmap
+            - cluster-service-configmap
+            - credentiald-configmap
+            - desmotes-configmap
```

Comment:
is this still a thing?

Reply:
Frankly speaking, I don't know. It was deployed into CP so I'm taking currently running state